graphql/spicepod.yaml

```diff
@@ -0,0 +1,30 @@
+version: v1beta1
+kind: Spicepod
+name: graphql_quickstart
+datasets:
+  - from: graphql:https://api.github.com/graphql
+    name: repos
+    params:
+      auth_token: [auth_token]
+      json_path: data.viewer.repositories.nodes
+      query: |
+        {
```

Comment:
Does this work with both query {} and without?  If so, let's note that in this quickstart.

Reply:
It's works the same way as in any other graphql interface